### PR TITLE
Add blogpost for forthcoming Packit 1.0.0

### PR DESCRIPTION
**Pull request by majamassarini**

### CODE REVIEW

File: posts/packit_1_0_0_action_required/index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  - configuration
+---
+
+After 105 minor releases we are almost ready to announce our first major release, Packit 1.0.0!
+
+<!--truncate-->
+
+## What's New
+
+We will **remove all since long deprecated keys from the configuration file**.
```
Comment (lbarcziova) [inline posts/packit_1_0_0_action_required/index.md:15]:
```suggestion
We will **remove all long-deprecated keys from the configuration file**.
```

sounds a bit more natural to me

File: posts/packit_1_0_0_action_required/index.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`packit-service` jobs are tight with `packit CLI` so be aware that for those projects still using deprecated keys in their `packit.yaml` project file we have created a migration pull request.
```
Comment (nforro) [inline posts/packit_1_0_0_action_required/index.md:17]:
```suggestion
`packit-service` jobs are tied with `packit CLI` so be aware that for those projects still using the deprecated keys in their `packit.yaml` configuration file we have created a migration pull request.
```

File: posts/packit_1_0_0_action_required/index.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Please, **check if you have any new PR for your project created by us, review and merge it**. Otherwise, you might experience issues with the `packit-service` jobs once the new packit version is released.
```
Comment (lbarcziova) [inline posts/packit_1_0_0_action_required/index.md:19]:
```suggestion
Please, **check if you have any new PR for your project created by us, review and merge it**. Otherwise, you might experience issues with the `packit-service` jobs once the Packit 1.0.0 is released.
```

I would also link the user that will create the PRs in the [by us]

File: posts/packit_1_0_0_action_required/index.md (hunk 2 above)
Comment (lbarcziova) [inline posts/packit_1_0_0_action_required/index.md:17]: I would maybe rephrase the beginning a bit, telling that the code regarding the configuration is placed in the `packit` package, therefore `packit-service` users will be affected as well.

File: posts/packit_1_0_0_action_required/index.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+We plan to release Packit 1.0.0 on **January 2025**.
```
Comment (nforro) [inline posts/packit_1_0_0_action_required/index.md:21]:
```suggestion
We plan to release Packit 1.0.0 in **January 2025**.
```

File: posts/packit_1_0_0_action_required/index.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+### List of removed keys
+
+- Job `build` does not exist anymore, changed in `copr_build`.
```
Comment (lbarcziova) [inline posts/packit_1_0_0_action_required/index.md:25]:
```suggestion
- Job `build` does not exist anymore, changed to `copr_build`.
```

**Review by lbarcziova — COMMENTED** [review]
:partying_face:

File: posts/packit_1_0_0_action_required/index.md (hunk 3 above)
Comment (majamassarini) [reply posts/packit_1_0_0_action_required/index.md:19]: I am using the packit-service tokens, see [here](https://gitlab.com/packit-service/hello-world/-/merge_requests/1298). So the user is Packit Service. Should I use different tokens? Probably I will need to create them for a different user in all gitlab.org instances...

Comment (lbarcziova) [reply posts/packit_1_0_0_action_required/index.md:19]: great, no, I think using the Packit Service user makes sense, so maybe just mention that explicitly (`created by the packit-service bot account`)

**Review by lbarcziova — APPROVED** [review]
thank you!

File: posts/packit_1_0_0_action_required/index.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+Please, **check if you have any new PR for your project created by the packit-service bot account, review and merge it**. Otherwise, you might experience issues with the `packit-service` jobs once the new packit version is released.
+
+We plan to release Packit 1.0.0 in **January 2025**.
+
+### List of removed keys
+
+- Job type `build` has been changed to `copr_build`.
+- Job type `production_build` has been changed to `upstream_koji_build`.
+- Key `upstream_project_name` has been changed to `upstream_package_name`.
+- Key `synced_files` has been changed to `files_to_sync`.
```
Comment (lachmanfrantisek) [inline posts/packit_1_0_0_action_required/index.md:29]: Do we want to mention a difference in behaviour? Or, at least link docs?

File: posts/packit_1_0_0_action_required/index.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+So be aware that for those projects still using the deprecated keys in their `packit.yaml` configuration file we have created a migration pull request.
+
+Please, **check if you have any new PR for your project created by the packit-service bot account, review and merge it**. Otherwise, you might experience issues with the `packit-service` jobs once the new packit version is released.
+
+We plan to release Packit 1.0.0 in **January 2025**.
+
+### List of removed keys
+
+- Job type `build` has been changed to `copr_build`.
+- Job type `production_build` has been changed to `upstream_koji_build`.
```
Comment (lachmanfrantisek) [inline posts/packit_1_0_0_action_required/index.md:27]: We can mention that this is for explicitness after introducing more build types.

**Review by lachmanfrantisek — APPROVED** [review]
:tada: Thanks Maja for working on all this!

File: posts/packit_1_0_0_action_required/index.md (hunk 6 above)
Comment (majamassarini) [reply posts/packit_1_0_0_action_required/index.md:29]: I will link the docs

**Comment by majamassarini** [comment]
recheck

**Comment by majamassarini** [comment]
recheck
